shared/cmd/flags.go

```diff
@@ -111,4 +111,16 @@ var (
 		Usage: "Specifies the upper limit of goroutines running before a status check fails",
 		Value: 5000,
 	}
+	// LogFileName specifies the log output file name.
+	LogFileName = cli.StringFlag{
+		Name:  "log-file",
+		Usage: "Specify log file name, relative or absolute",
+		Value: "text",
```

Comment:
`Value` is the default value, lets omit this for `log-file`

Reply:
Done.